dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.certificateCreator-2.0.feature

```diff
@@ -6,5 +6,5 @@ Subsystem-Version: 2.0.0
 IBM-App-ForceRestart: install, \
  uninstall
 -bundles=com.ibm.ws.crypto.certificate.creator.acme
-kind=noship
+kind=ga
 edition=full
```

Comment:
full probably isn't the right edition?

Reply:
It is not! We are reviewing the correct edition.

Reply:
Changed to base.